Automattic/wp-calypso
# Sites dashboard: Add a feature family bar just with Overview tab initially

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/67

## Proposed Changes

* This PR adds a feature family bar, only with the Overview tab for now.

## Testing Instructions

* To test, apply this PR and then run `yarn start-a8c-for-agencies`. Next, visit `http://agencies.localhost:3000/sites`. You should see a feature family bar showing 'Overview' as the only item, selected.
* Open the preview pane, and the bar should disappear.

Overview item displaying:

<img width="958" alt="Screenshot 2024-03-15 at 10 52 16" src="https://github.com/Automattic/wp-calypso/assets/16754605/54ca98e3-b87e-4db7-9f9f-d22fbcccc300">


No feature family bar when the preview pane is open:

<img width="1365" alt="Screenshot 2024-03-14 at 13 57 49" src="https://github.com/Automattic/wp-calypso/assets/16754605/4badc38b-46ca-414d-86c3-b497642d4348">

## Pre-merge Checklist
